src/simulation_lifecycle/simulation.c

```diff
@@ -89,3 +92,58 @@ int write_sim_config(const cJSON *simulation_config, char *config_json) {
     }
     return write_data_to_file(cJSON_GetStringValue(config_output_path), config_json);
 }
+
+int run_sim(const cJSON *simulation_config){
+
+    /* Get model name - e.g. 2_2_agent_sir_config */
+    cJSON *model_id = cJSON_GetObjectItemCaseSensitive(simulation_config, SIM_MODEL_ID);
+    char *model = cJSON_GetStringValue(model_id);
+
+    /* Define executable - e.g. 2_2_agent_sir_config.exe */
+    char *model_executable = concat(model, ".exe");
```

Comment:
This is only for Windows. In MacOS, for instance, the binary has no extension at all.

We must add a macro definition, something like:
```
#ifdef WIN32
    model = concat(model, ".exe");
#endif
```

Reply:
In the Cygwin terminal or the Windows Command Prompt, inserting ".exe" or not works. So I can just remove the extension. However, the Windows Command Prompt, won't take "./". Do we want to be able to use the Windows Command Prompt or just the Cygwin terminal?

Reply:
How does it work in the Windows command prompt?

If it is easy to adapt, we can try

Reply:
Should be easy to adapt. I'll look into it.